alune/config.py

```diff
@@ -121,3 +124,26 @@ def get_traits(self) -> list[images.Trait]:
             A list of traits we look for.
         """
         return self._config["traits"]
+
+    def get_auto_surrender(self) -> bool:
+        """
+        Get the surrender option the user wants.
+
+        Returns:
+            A Boolean that determines whether or not we should surrend when possible.
+        """
+        return self._config["auto_surrender"]
+
+    def get_auto_surrender_random_delay(self) -> int:
+        """
+        Get the surrender delay
+
+        Returns:
+            An random Integer between [1 and auto_surrender_random_delay]
+            Returns 0 if feature disabled or negative value.
+        """
+        return (
+            0
+            if self._config["auto_surrender_random_delay"] is None or self._config["auto_surrender_random_delay"] <= 0
+            else _random.randint(1, self._config["auto_surrender_random_delay"])
+        )
```

Comment:
No point in an inline statement if it's longer and harder to read than just writing it out:

```py
delay_upper_bound = self._config["surrender_delay"] or 0
if delay_upper_bound <= 0:
    return 0
return _random.randint(1, delay_upper_bound)
```